### ISSUE TITLE
Feat: improve message toast's text 

### ISSUE THREAD

**Issue by AnthonyNahas** [AnthonyNahas/ngx-auth-firebaseui#163]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```

### Versions
`angular v7.1.4`

### Desired functionality

The developers should be able to interpolate the toast message input `messageOnAuthSuccess ` with `user.displayName`

### Mention any other details that might be useful
related to #156 - thanks for @Avrohom

**Comment by Avrohom** [comment]
👍

**Comment by AnthonyNahas** [comment]
This feature is already so far and pushed! It will be available in the next minor release (v2.3.0 which is planned this week or maybe the next few days..). 

You will able to achieve that like the following e.g:


#### html

```html
<ngx-auth-firebaseui
          (onSuccess)="saveUser($event)"
          (onError)="handleError($event)"
          [messageOnAuthSuccess]="message"
          [messageOnAuthError]="errorMessage">
</ngx-auth-firebaseui>
```

#### Typescript

```typescript
import {Component} from '@angular/core';

@Component({
  selector: 'app-messages',
  templateUrl: './messages.component.html',
  styleUrls: ['./messages.component.scss']
})
export class MessagesComponent {

  user: any;

  message: string;
  errorMessage: string;

  saveUser($event) {
    this.user = $event;
    this.message = `${this.user.displayName} here we go!`;
    console.log('Auth success - user = ', this.user, this.user.displayName);
  }

  handleError($event) {
    this.errorMessage = `Oops! ${$event}`;
  }
}
```

I will ping you here when the update is live

cheers 🍻

**Comment by AnthonyNahas** [comment]
[v2.3.0](https://www.npmjs.com/package/ngx-auth-firebaseui) is live now! thanks for contributing @Avrohom  🍺
